index.html

```diff
@@ -9,29 +9,46 @@
   <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
   <link href="css/index.css" rel="stylesheet">
 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
 
- <!-- navigation code here -->
+ <div class="container navigation">
+   <h1>SpaceWalkers</h1>
+   <nav>
+     <ul>
+       <li><a href="">Home</a></li>
+       <li><a href="">About Us</a></li>
+       <li><a href="">Blog</a></li>
+       <li><a href="">Contact</a></li>
+       <li><a href="">Sign In</a></li>
+     </ul>
+   </nav>
+ </div>
 
   <div class="container home">
 
     <header class="intro">
-       <!-- image code here -->
+       <div class="header-img">
+         <img src="https://github.com/Angie0488/Sprint-Challenge--Advanced-CSS/blob/master/img/main-header.png?raw=true" alt="main-header image">
```

Comment:
This is incorrect image source syntax. It should look something like this instead:
`src="img/main-header.png"`
Even if it works in this specific case, if you were to upload your images and page elsewhere and remove your github repository, this would cause your page to break and not render as intended.